src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -137,11 +136,11 @@ private async Task<Build> GetRootBuildAsync()
                     return null;
                 }
                 Console.WriteLine($"Looking up builds of {_options.RepoUri}@{_options.Commit}");
-                IList <Build> builds = await remote.GetBuildsAsync(_options.RepoUri, _options.Commit);
+                IEnumerable<Build> builds = await remote.GetBuildsAsync(_options.RepoUri, _options.Commit);
                 // If more than one is available, print them with their IDs.
-                if (builds.Count > 1)
+                if (builds.Count() > 1)
```

Comment:
>.Count() [](start = 26, length = 8)

If there were a lot of builds this would hit perf since it needs to go through all the items to get the count. Since I don't expect the builds amount to be huge I don't thing this is a big concern

Reply:
I think this depends on what the backing collection for the IEnumerable actually is.  In this case, I think it's an IList, which I believe will be constant time Count().  Anyways, it's fine.